.travis.yml

```diff
@@ -15,6 +15,8 @@ install:
 before_script:
     - yarn flow
     - yarn lint
+
+script:
```

Comment:
If test break on `script`, will it process `after_script`?

Reply:
[According to this doc](https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build), I believe it's “yes”.
Or I was thinking to move `coverall` to `after_success` so it only runs after tests passed?